src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Views/StudyResultView.sql

```diff
@@ -0,0 +1,32 @@
+﻿/*************************************************************
+    Study View to be used for getting computed column response
+**************************************************************/
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.views
+    WHERE   Name = 'StudyResultView'
+)
+BEGIN
+    EXEC('CREATE VIEW dbo.StudyResultView
+    WITH SCHEMABINDING
+    AS
+    SELECT  st.StudyInstanceUid,
+            st.PatientId,
+            st.PatientName,
+            st.ReferringPhysicianName,
+            st.StudyDate,
+            st.StudyDescription,
+            st.AccessionNumber,
+            st.PatientBirthDate,
+            (SELECT STRING_AGG(Modality, '','')
+            FROM dbo.Series se 
+            WHERE st.StudyKey = se.StudyKey
+            AND st.PartitionKey = se.PartitionKey) AS ModalitiesInStudy,
+            (SELECT SUM(1) 
+            FROM dbo.Instance i 
+            WHERE st.StudyKey = i.StudyKey) AS NumberofStudyRelatedInstances,
```

Comment:
Do we want to have all instances in a partition?

Reply:
StudyKey is unique across partition. I had the partitionKey filter initially, but there is no matching index.

I added in the comments for future. I will create another PR to fix all the indexes and some allinment on when to use UIDs vs keys vs Partition. There are some missing indexes for the already existing QIDO queries.